pipeline/seed_data.py

```diff
@@ -0,0 +1,133 @@
+"""Small example of connecting to an MSSQL database."""
+import csv
+from os import environ
+
+from dotenv import load_dotenv
+from sqlalchemy import create_engine, sql, Connection
+
+from transform import standardize_country_name
+
+
+def get_connection()-> Connection:
```

Comment:
Quite nit picky (said the same to Charlie) - but but maybe think about passing in a `config` argument here (so replace `environ` in this function with `config`, and then pass in environ in the `if __name__ == "__main__"` statement - just a thought I had, but not crucial to change by any means and probably just a stylistic choice!